docs/attributes-registry/datacenter.md

```diff
@@ -0,0 +1,24 @@
+<!--- Hugo front matter used to generate the website version of this page:
+--->
+
+<!-- NOTE: THIS FILE IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/attribute_namespace.md.j2 -->
+
+# Datacenter
+
+## Datacenter Attributes
```

Comment:
We have tooling to autogenerate markdown, you should use `<!-- semconv datacenter -->` here for consistency.

See: https://github.com/open-telemetry/semantic-conventions/blob/main/model/README.md

Reply:
🤔 I used the automated tool. I wonder if because the name was missing this resulted.

Reply:
Well I found out what the issue was, perhaps I missed it in my reading of the various documentation, but I didn't see any mention that you need to first manually create a file which has the reference which you are mentioning, and then it will replace the contents between. I can understand the use, but it's not exactly clear as a first time user of the weaver tool.

Reply:
@jsuereth am i missing something

Reply:
@hdost Sorry for radio silence here. You had fixed this earlier, forgot to mark this as closed.  Apologies the docs aren't intuitive, feel free to open suggestions so we can fix them.